src/foremast/consts.py

```diff
@@ -197,10 +197,15 @@ def _generate_security_groups(config_key):
 DOMAIN = validate_key_values(CONFIG, 'base', 'domain', default='example.com')
 ENVS = set(validate_key_values(CONFIG, 'base', 'envs', default='').split(','))
 REGIONS = set(validate_key_values(CONFIG, 'base', 'regions', default='').split(','))
+MANUAL_PIPELINE_SYNONYMS = set(
+    validate_key_values(CONFIG, 'base', 'manual_pipeline_synonyms', default='manual').split(','))
 ALLOWED_TYPES = set(
-    validate_key_values(CONFIG, 'base', 'types', default='ec2,lambda,s3,datapipeline,rolling,manual').split(','))
+    validate_key_values(CONFIG, 'base', 'types', default='ec2,lambda,s3,datapipeline,rolling,manual')
```

Comment:
Don't think we need to append `ALLOWED_TYPES` here. Since we are extending `manual` only with this PR, think we can assume if users allow the use of `"manual"` pipelines, they will also get access to various names.

We can do the check of manual pipelines names later in the logic.

Reply:
Remove the `|` and we can do the logic later.